aws-marketplace/licensemanager/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.13-alpine
```

Comment:
Let's stick with one name everywhere, either `licensemanager` or `license-manager`.

Reply:
I think you're confusing a couple usages. When I grep for `license-manager`, I find the following:

```
aws-marketplace/licensemanager/checkoutLicense.py
7:client = boto3.client('license-manager', region_name='us-east-1')

aws-marketplace/TESTING.md
51:erform: license-manager:CheckoutLicense because no identity-based policy allows the license-manager:CheckoutLicense action 
58:This means that the NodeGroup needs the license-manager:CheckoutLicense permission. Contact IT to grant this permission.
```

the first usage refers to the AWS API. This isn't something I named.  The 2nd and 3rd usages refers to the AWS API permissions.


Everywhere else, we use `licensemanager` which refers to the name of the docker image.

Reply:
@jessesimpson36 Any reason to have that distinguish? Cannot we simply name the Docker image `license-manager`?

Reply:
I renamed this to license-manager